Pipfile.lock

```diff
@@ -884,62 +928,66 @@
         },
         "typing-extensions": {
             "hashes": [
-                "sha256:6e95524d8a547a91e08f404ae485bbb71962de46967e1b71a0cb89af24e761c5",
-                "sha256:79ee589a3caca649a9bfd2a8de4709837400dfa00b6cc81962a1e6a1815969ae",
-                "sha256:f8d2bd89d25bc39dabe7d23df520442fa1d8969b82544370e03d88b5a591c392"
+                "sha256:7cb407020f00f7bfc3cb3e7881628838e69d8f3fcab2f64742a5e76b2f841918",
+                "sha256:99d4073b617d30288f569d3f13d2bd7548c3a7e4c8de87db09a9d29bb3a4a60c",
+                "sha256:dafc7639cde7f1b6e1acc0f457842a83e722ccca8eef5270af2d74792619a89f"
             ],
-            "markers": "python_version < '3.7'",
-            "version": "==3.7.4.2"
+            "markers": "python_version < '3.8' and python_version < '3.7'",
```

Comment:
I don't pretend to understand this entirely but if something is < 3.7 then it's automatically < 3.8 so should this just be:
```suggestion
            "markers": "python_version < '3.7'",
```

Reply:
The lock file is automatic, and I have no control over the contents of it. It shouldn't be edited manually.
I'd wager that, instead of being smart, pipenv simply combines the requirements of all sub-dependencies together into one statement. The overall effect is still valid.

Reply:
thanks - makes sense